snapshot/lib/snapshot/collector.rb

```diff
@@ -7,8 +7,15 @@ def self.fetch_screenshots(output, dir_name, device_type, launch_arguments_index
       containing = File.join(TestCommandGenerator.derived_data_path, "Logs", "Test")
       attachments_path = File.join(containing, "Attachments")
 
-      to_store = attachments(containing)
-      matches = output.scan(/snapshot: (.*)/)
+      language_folder = File.join(Snapshot.config[:output_directory], dir_name)
+      FileUtils.mkdir_p(language_folder)
+
+      if Snapshot.config[:simultaneous] || Helper.xcode_at_least?(9)
+        return collect_screenshots_for_language_folder(language_folder)
+      else
+        to_store = attachments(containing)
```

Comment:
Maybe add a few lines of comment on why this `if else` block exists, and how the plist format has changes with this release